_robots/2022Robot.md

```diff
@@ -14,7 +14,7 @@ features:
       icon: fa-robot
     - label: ~95% high goal shooting accuracy
       icon: fa-crosshairs
-rating: 2
+rating: 4
```

Comment:
Can we remove the `<!--price ... -->` comment from these files?
What does the `rating` attribute mean in these files?

Should the bulleted lists below be unordered lists? ex.
```
<h2>Robot capabilities</h2>
<ul>
  <li>Four wheel...</li>
  ...
</ul>
```